workers/unity/Packages/com.improbable.gdk.core/Worker/IUpdateDiffStorage.cs

```diff
@@ -0,0 +1,55 @@
+using Improbable.Gdk.Core.Commands;
+
+namespace Improbable.Gdk.Core
+{
+    public interface IComponentDiffStorage
+    {
+        uint GetComponentId();
+
+        void Clear();
+        void RemoveEntityComponent(long entityId);
+    }
+
+    public interface ICommandDiffStorage
+    {
+        uint GetComponentId();
+        uint GetCommandId();
+
+        void Clear();
+        void RemoveEntityComponent(long entityId);
```

Comment:
Seems like this was a copy-paste job and forgot to change this name. 

I wouldn't know what this method is meant to do on an `ICommandDiffStorage`

Reply:
Was named as it removes stuff related to that entity component. Renamed to RemoveRequests